ui/src/wide_gauge_stat.tsx

```diff
@@ -76,8 +73,19 @@ interface State {
 
 export const
   View = bond(({ name, state: s, changed }: Model<State>) => {
+    const
+      valueNonDigitCharCount = substitute(s.value, s.data).replace(/[0-9]/g, '').length,
+      auxValueNonDigitCharCount = substitute(s.aux_value, s.data).replace(/[0-9]/g, '').length
     const render = () => {
-      const data = unpack<Rec>(s.data)
+      const
+        data = unpack<Rec>(s.data),
+        /*
+         * Numbers 14 and 9 are representing maximum width of the single character.
+         * The minimum width of the value containers is the width of 4 characters.
+         * This prevents the jumping layout in the most common scenarios when numbers are changing between ones to tens. 
+        */
+        valueContainerWidth = 14 * Math.max(substitute(s.value, s.data).length, 4 + valueNonDigitCharCount),
+        auxValueContainerWidth = 9 * Math.max(substitute(s.aux_value, s.data).length, 4 + auxValueNonDigitCharCount)
```

Comment:
Let's substitute only once, put the values into variables and use the variables for both digitcounts and containerwidths.

Reply:
The problem is that in that case digitcounts, especially `.replace(/[0-9]/g, '').length` part will be computed on every render what is not necessary. Are you sure you want it like this?

Reply:
Sure. Wouldn't doing otherwise cause a bug? After updating the value, the digit counts would not be updated thus be out of date.

Reply:
Nope, because `valueNonDigitCharCount` and `auxValueNonDigitCharCount` says about the number of non-digit characters which is always the same. Let's say you have value defined in this format: `value='=${{intl foo minimum_fraction_digits=2 maximum_fraction_digits=2}} (USD)'`
So the real example would be: `$43.02 (USD)`
That means you have 8 non-digits character here: `$.[space](USD)`
Even if value changes to `$143.02 (USD)`or `$3.12 (USD)` the number of non-digit characters does not change. Therefore it is fine to have it computed just once.

Reply:
What about changing the value from `1 share` to `2 shares`?